Formula/med-file.rb

```diff
@@ -0,0 +1,48 @@
+class MedFile < Formula
+  desc "Modeling and Data Exchange standardized format"
+  homepage "http://www.salome-platform.org"
+  url "http://files.salome-platform.org/Salome/other/med-3.2.0.tar.gz"
+  sha256 "d52e9a1bdd10f31aa154c34a5799b48d4266dc6b4a5ee05a9ceda525f2c6c138"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "swig" => :build
+  depends_on "hdf5"
+  depends_on "python"
+
+  patch do
```

Comment:
new formulae need to build without patching

Reply:
These are legacy patches that already exist in the patch repo and were previously upstream.  Just waiting for them to address the patches and I will remove them.

Reply:
New version was released (3.2.1), hopefully allows to remove the patch